src/fabtrimino.c

```diff
@@ -0,0 +1,250 @@
+#include "fabtrimino.h"
+
+struct square
+{
+    enum shape shape;
+};
+
+struct fabtrimino
+{
+    enum shape shape;
+    struct square*** matrix;
+    struct vector* pos;
```

Comment:
Not sure why this is heap allocated since it's not actually being used. Pretty sure this could just be statically allocated as you probably only have one of them.